extensions/pcap/CMakeLists.txt

```diff
@@ -19,11 +19,19 @@
 
 find_package(PCAP REQUIRED)
 
+include(FetchContent)
+
+FetchContent_Declare(pcapplusplus
+		URL      https://github.com/seladb/PcapPlusPlus/archive/refs/tags/v21.05.tar.gz
+		URL_HASH SHA256=f7bc2caea72544f42e3547c8acf65fca07ddd4cd45f7be2f5132dd1826ea27bb
+		)
+FetchContent_MakeAvailable(pcapplusplus)
+
 include(${CMAKE_SOURCE_DIR}/extensions/ExtensionHeader.txt)
 
 include_directories(/usr/include/netinet)

```

Comment:
Could you convert the `include_directories` calls to target-oriented equivalents? If you don't mind fixing this old issue.